docs/en/siem/cases-api-add-comment.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[cases-api-add-comment]]
+=== Add comment
+
+Adds a comment to an existing case.
+
+==== Request URL
+
+`POST <kibana host>/<port>/api/cases/<case ID>/comments`
+
+===== URL parts
+
+The URL must include the `case ID` of the case to which you are adding a 
+comment. Call <<cases-api-find-cases>> to retrieve case IDs.
+
+==== Request body
+
+A JSON object with a `comment` field:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comment` |String |The case's new comment. |Yes
+|==============================================
+
+===== Example request
+
+Adds a comment to case ID `a18b38a0-71b0-11ea-a0b2-c51ea50a58e2`:
+
+[source,sh]
+--------------------------------------------------
+POST api/cases/a18b38a0-71b0-11ea-a0b2-c51ea50a58e2/comments
+{
+  "comment": "That is nothing - Ethan Hunt answered a targeted social
+  media campaign promoting phishy pension schemes to IMF operatives."
+}
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+   Indicates a successful call.
+
+==== Response payload
+
+A JSON object that includes the user who created the comment, and the comment's
+ID, version, and creation time.
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
```

Comment:
this now returns the full case the comment was posted to with `comments=true`. Example:

```
{
  "id": "9d386a20-74ee-11ea-bee8-a9c66969e1b5",
  "version": "WzYzNCwxXQ==",
  "comments": [
    {
      "id": "cee89950-74ee-11ea-bee8-a9c66969e1b5",
      "version": "WzYzMywxXQ==",
      "comment": "Solve this fast!",
      "created_at": "2020-04-02T14:32:41.016Z",
      "created_by": {
        "email": "Steph.milovic@elastic.co",
        "full_name": "Steph Milovic",
        "username": "smilovic"
      },
      "pushed_at": null,
      "pushed_by": null,
      "updated_at": null,
      "updated_by": null
    }
  ],
  "totalComment": 0,
  "description": "This looks not so good",
  "title": "Bad meanie defacing data",
  "tags": [
    "defacement"
  ],
  "closed_at": null,
  "closed_by": null,
  "created_at": "2020-04-02T14:31:17.655Z",
  "created_by": {
    "email": "Steph.milovic@elastic.co",
    "full_name": "Steph Milovic",
    "username": "smilovic"
  },
  "external_service": null,
  "status": "open",
  "updated_at": "2020-04-02T14:32:41.016Z",
  "updated_by": {
    "username": "smilovic",
    "full_name": "Steph Milovic",
    "email": "Steph.milovic@elastic.co"
  }
}
```